contracts/SmartVaultYieldManager.sol

```diff
@@ -185,15 +188,16 @@ contract SmartVaultYieldManager is ISmartVaultYieldManager, Ownable {
         if (_hypervisorData.hypervisor != _hypervisor) revert InvalidRequest();
         IHypervisor(_hypervisor).withdraw(_thisBalanceOf(_hypervisor), address(this), address(this), [uint256(0),uint256(0),uint256(0),uint256(0)]);
         _swapToSingleAsset(_hypervisor, _token, uniswapRouter, _hypervisorData.poolFee);
-        IERC20(_token).safeTransfer(msg.sender, _thisBalanceOf(_token));
     }
 
     function withdraw(address _hypervisor, address _token) external {
         IERC20(_hypervisor).safeTransferFrom(msg.sender, address(this), IERC20(_hypervisor).balanceOf(msg.sender));
```

Comment:
super minor, but code might become much more readable with interface wrappings being class defined instead of the member always being wrapped multiple times in every function call.